applications/views.py

```diff
@@ -20,6 +24,77 @@ def validate_application_access(user, application):
     raise Http404
 
 
+class ResearcherCreate(CreateView):
+    form_class = ResearcherRegistrationForm
+    model = ResearcherRegistration
+    template_name = "applications/researcher_form.html"
+
+    def dispatch(self, request, *args, **kwargs):
+        self.application = get_object_or_404(Application, pk=self.kwargs["pk"])
+
+        validate_application_access(request.user, self.application)
+
+        return super().dispatch(request, *args, **kwargs)
+
+    def form_valid(self, form):
+        researcher = form.save(commit=False)
+        researcher.application = self.application
+        researcher.save()
+
+        # TODO: how should we configure this so it's not hardcoded?
```

Comment:
Change `page_num` to `key`, and then use descriptive strings for the keys in the form spec?  A small amount of work would be required (probably in `wizard.py`) to get the next page.